sdk/src/main/java/com/relaypro/sdk/Relay.java

```diff
@@ -241,10 +275,10 @@ private MessageWrapper sendRequest(Map<String, Object> message, boolean waitForP
      * @param target the device that you would like to start an interaction with.
```

Comment:
If we were to move forward with this change, in all the places where we document the target parameter, we'll need to either describe or point to a description of all the acceptable possibilities for this value.